mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -402,6 +402,35 @@ def push_boefje_task(
             task_db.status = models.TaskStatus.FAILED
             self.ctx.datastores.task_store.update_task(task_db)
 
+        # Is the ooi in other organisations? When the ooi is in other
+        # organisations we need to create tasks for those organisations as well.
+        if boefje_task.input_ooi is not None:
+            # FIXME: is the ooi shared between organisations, or is it a copy?
+            # If it is a copy we need to use the ooi id in the boefje task.
+            orgs = self.is_ooi_in_other_organisations(boefje_task.input_ooi)
+            if orgs:
+                boefje_task.organizations = orgs
+
+                # FIXME: what if boefje is disabled in the other org? Are we sure
+                # that in that org the boefje is allowed to scan the ooi?
+                # FIXME: what status do we give it do we give it completed, we
+                # can't make it queued since it shouldn't be picked up by
+                # the task runner.
+                # FIXME: how to these tasks get associated with the other raw files?
```

Comment:
I think we discusses to upload the raw files for each job, and as such we'd create normalizer jobs in each subsequent organizaton (which we might deduplicate later on).
Uploaded raw files with the same content can transparently be deduplicated (by content hash) in Bytes  on disk.